### ISSUE TITLE
Metadata ETS: md-iop.a.3 Encoding should not require specification

### ISSUE THREAD

**Issue by MarieLambois** [inspire-eu-validation/ets-repository#30]
Only name and version are required so this should not be prensent: 
The metadata record set has 1 record(s) with errors for this assertion.
XML document 'IGNF_BDTOPOr_2-2.xml', record 'IGNF_BDTOPOr_2-2.xml': Every metadata record must document the encoding in which the dataset is available and specify the name, version and  specification of the encoding, but this record does not have the 'specification' property.

**Comment by cportele** [comment]
Then text in the data specifications would be incorrect:

_TG Requirement 4: Metadata instance (XML) documents shall contain the elements and meet the INSPIRE multiplicity specified in the sections below._

and

_The property values (name, version, specification) [...] shall be used to document the default and alternative encodings._

seem to indicate that the **specification** element has to be provided, too. 

This would also be consistent with the information provided for each encoding in the Clauses 9 of the data specifications. For example:

_9.3.1.2. Default encoding for application schema AdministrativeUnits

Name: AdministrativeUnits GML Application Schema
Version: version 3.1
Specification: D2.8.I.4 Data Specification on Administrative Units – Technical Guidelines_

**Comment by MarieLambois** [comment]
Good point. I only refered to the TG (2.0 since it is interoperability metadata) and they only mention name and version (see C.3.3)

![image](https://cloud.githubusercontent.com/assets/12797943/22779658/f7b4ac5a-eebb-11e6-8fa0-db91b78c3249.png)

**Comment by cportele** [comment]
Ah, interesting. So the requirements in the data specifications on interoperability metadata differ from those in the (new) metadata technical guidance.

The current Executable Test Suite refers to the requirements in the data specifications (via http://inspire.ec.europa.eu/id/ats/data/3.0rc3/interoperability-metadata). Therefore, it is correct that the current Executable Test Suite requires the specification element.

When the Abstract Test Suite and the Executable Test Suites for the new metadata technical guidance are added, then there should be no test on the specification element.

@michellutz :

The guidance on the use of the name and version elements seems to have changed with the new metadata technical guidance, too. So far it was based on the names and versions specified in the data specifications:

_Name: AdministrativeUnits GML Application Schema_
_Version: 3.1_

Now, the example seems to imply to just use:

_Name: GML_
_Version: 3.2.1_

or the use of the media types. Both would not be consistent with the Clauses 9 in the data specifications. Will the data specifications will be updated to reflect the change in approach?

**Comment by michellutz** [comment]
Sorry for getting back to this only now. I think the issues raise the larger question on how we deal with issues coming from feedback on the validator/ETS, which are not ETS-implementation related, but related to issues in the ATS or even TGs. I think there could be a role for a revived MIWP-5 sub-group that would collect and discuss such issues and would elaborate change requests to ATS/TGs to be reviewed and endorsed by the MIG (similar to the way the MIWP-14 sub-group works with change requests coming from the Thematic Clusters). We will discuss this at the upcoming MIG-T face-to-face meeting.

I will also provide my own view on the issues - but after I'm back from my holidays (13 March).

**Comment by heidivanparys** [comment]
@michellutz I think this issue should have the "for-2017.4-discussion" label.

**Comment by michellutz** [comment]
I think we are relatively free to decide one way or the other - we should discuss what makes most sense from a user point of view.

The IR on data interoperability require a MD element on "Encoding: Description of the computer language construct(s) specifying the representation of data objects in a record, file, message, storage device or transmission channel."

I think being more specific (i.e. specify a concrete encoding rather than just the underlying fromat) will be more useful for being able to use/interpret the data. But then again, more detailed information can be obtained from the download service's Describe Spatial Dataset or Describe Spatial Object operations, so maybe the format is enough for the discovery metadata.

Once decided, we should then consistently apply the rule across specs, ATS & ETS.

**Comment by michellutz** [comment]
[2017.4 meeting 2018-03-16] It was agreed to document the underlying format (using a name and a version), not the concrete application schema in the "Encoding" metadata element (e.g. name: GML, version: 3.2.1), but to also recommend using a reference to the relevant media-type (available in the INSPIRE registry at http://inspire.ec.europa.eu/media-types) in an Anchor element (e.g. https://www.iana.org/assignments/media-types/application/gml+xml for GML).

The MD TG v2.0, the ATS and ETS need to be updated accordingly.

**Comment by michellutz** [comment]
Looking at the TG MD v2.0, I noticed that section 3.2.3.1 Data encoding is actually consistent with the data specification template, at least in the Example 3.18, but this is then inconsistent with the Annex C3.3. Based on the agreed change, we need to therefore also update section 3.2.3.1.

**Comment by michellutz** [comment]
[2017.4 meeting 2018-04-19] Keep the issue open, since it might still require some work on ATS/ETS. The proposed TG corrigenda are tracked on the MIG collaboration platform in https://ies-svn.jrc.ec.europa.eu/issues/3217

**Comment by inakidiazdecerio** [comment]
Dear @michellutz ,
As we agreed in the last meeting we are going to progress on developments / improvements on the ETS and ATS while the technical guidance is modified.

From our understanding of the issue: name, version and specification (new!) should be checked for distributionFormat element.

**Comment by michellutz** [comment]
Yes, only name and version should be available. Specification should not be checked any more.
